service/collector/app_resource.go

```diff
@@ -115,7 +118,11 @@ func (c *AppResource) Describe(ch chan<- *prometheus.Desc) error {
 }
 
 func (c *AppResource) collectAppStatus(ctx context.Context, ch chan<- prometheus.Metric) error {
-	r, err := c.g8sClient.ApplicationV1alpha1().Apps("").List(metav1.ListOptions{})
+	options := metav1.ListOptions{
+		LabelSelector: key.AppVersionSelector(c.uniqueApp).String(),
```

Comment:
I'm searching for how to use selector directly inside k8sclient, but looks like there's no way. I think it's rather simple.

Reply:
Yes I don't know of a better way to do this. Thanks for researching it but lets go with this.